### PR TITLE
Integrate GeniusAPI for the Lyrics command

### PR DESCRIPTION
**Pull request by Sebs5384**
This will setup a new functionality which makes requests to the Genius API by using either prefix or slash commands

### CODE REVIEW

File: commands/lyrics.js [hunk 1]
```diff
@@ -1,39 +1,164 @@
...
 export const command = new SlashCommandBuilder()
     .setName('lyrics')
     .setDescription('Asks Kermit to google for lyrics')
 
-command.aliases = ['l', 'lyric']
-
-command.slashRun = async function slashRun(client, interaction)
-{
-    await interaction.followUp({ content: 'Lets ask Kermit!', ephemeral: true });
+command.aliases = ['.l', '.lyric']
```
Comment (arminherling) [inline commands/lyrics.js]: You forgot to revert this change

File: commands/lyrics.js [hunk 2]
```diff
@@ -1,39 +1,164 @@
...
-    await run(client, channel, send)
+    await run(client, channel, send, member)
 }
 
-async function run(client, channel, send)
-{
+async function run(client, channel, send, member) {
+
+    const lyricsToken = client.lyricsToken
+    const geniusClient = new Genius.Client(lyricsToken)
```
Comment (arminherling) [inline commands/lyrics.js]: It looks like client.lyricsToken is never set, maybe you forgot to commit that.
Maybe we can set client.genius = new Genius.Client instead of the token, that might be nicer.

File: commands/lyrics.js [hunk 3]
```diff
@@ -1,39 +1,164 @@
...
+    const intersection = userSong.filter(songName => queueSong.includes(songName))
+    const similaritiesValue = intersection.length / Math.max(userSong.length, queueSong.length)
+
+    return similaritiesValue
+}
+
+async function handlePagination(client, message, member, songList, send, currentPage){
+    const commandAuthorFilter = i => i.user.id === member.id
+
+    const collector = message.createMessageComponentCollector({ commandAuthorFilter, time: 300000 })
```
Comment (arminherling) [inline commands/lyrics.js]: Could you add a comment that says how many second this is please?

File: commands/play.js [hunk 4]
```diff
@@ -1,5 +1,7 @@
 import { SlashCommandBuilder } from 'discord.js'
 
+export const userSongs = []
```
Comment (arminherling) [inline commands/play.js]: The bot can be connected to multiple voice channels and play different songs at the same time.
So instead of exporting this, we should try to put it on the queue because each vc has their own queue.

**Review by arminherling — COMMENTED** [review]
Looks good so far.
There are some things like formatting but thats not important. 
As a note for the future, it might be nice to combine createLyricsButtons and createPlaylistButtons in playlist.js into another function to reduce code duplication.

File: commands/lyrics.js (hunk 1 above)
Comment (Sebs5384) [reply commands/lyrics.js]: Reverted this change

File: commands/lyrics.js (hunk 3 above)
Comment (Sebs5384) [reply commands/lyrics.js]: Added a comment and turned it into a constant instead of just a magic number

File: commands/play.js (hunk 4 above)
Comment (Sebs5384) [reply commands/play.js]: Added a new key to the queue object and then used the previous exported array as its value to keep track of the userSongs, the playSong file will notice after the second song is being played this avoids an issue that comes down to playSong being executed first than play.js.

File: commands/lyrics.js (hunk 2 above)
Comment (Sebs5384) [reply commands/lyrics.js]: Added this change, it is now  client.genius rather than the constant geniusClient

**Comment by arminherling** [comment]
LGTM
